docs/api-guide/fields.md

```diff
@@ -290,9 +290,9 @@ A date and time representation.
 
 Corresponds to `django.db.models.fields.DateTimeField`.
 
-**Signature:** `DateTimeField(format=None, input_formats=None)`
+**Signature:** `DateTimeField(format=empty, input_formats=None)`
```

Comment:
Considering if we instead treat empty as an internal implementation detail, perhaps documenting the signature as `DATETIME_FORMAT`, instead?